lib/hammer_cli_foreman_remote_execution/options/normalizers.rb

```diff
@@ -1,3 +1,5 @@
+require 'parse-cron'
```

Comment:
Yea, it's used to do the validation, it lets us give the user some help and validate if the cron's valid before sending it off to the server, see below in the format method.